vagrant/provisioning/roles/microservices/tasks/main.yaml

```diff
@@ -0,0 +1,26 @@
+- name: Expose Microservices ports
+  become: yes
+  firewalld:
+    port: "{{ item }}"
+    permanent: true
+    state: enabled
+    immediate: yes
+  register: microservices_ports
+  loop:
+    - "9092/tcp"   # Kafka
+
+- name: Reload firewalld
+  become: yes
+  command: firewall-cmd --reload
+  when: microservices_ports is changed
```

Comment:
with `immediate: yes` you don't actually need to call `firewall-cmd --reload`.

Reply:
I have copied this from somewhere. And yes it is not needed.